bpmn2stamp-converter/src/main/java/cz/cvut/kbss/bpmn2stamp/converter/common/ApplicationConstants.java

```diff
@@ -43,6 +43,11 @@ public class ApplicationConstants {
      */
     public static final String ORG_GROUP_SUFFIX = "_group";
 
+    /**
+     * Composite id delimiter.
+     */
+    public static final String COMPOSITE_ID_DELIMITER = "_";
```

Comment:
```suggestion
    public static final String COMPOSITE_ID_DELIMITER = "-";
```
I believe we use "-" in most of the cases.